.travis.yml

```diff
@@ -2,18 +2,39 @@ sudo: false
 language: php
 php:
   - 7.2
-
-branches:
-  only: 
-    - master
+services:
+  - docker
+addons:
+  apt:
+    packages:
+      - docker-ce
+env:
+  - DOCKER_COMPOSE_VERSION=1.23.2
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - "$HOME/.npm"
 
 before_install:
-  - git config --global github.accesstoken $GITHUB_OAUTH_TOKEN
-  - composer config -g github-oauth.github.com $GITHUB_OAUTH_TOKEN --no-interaction
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  - nvm install 8
+  - nvm use 8
+
+install:
+  - npm ci
+  - ( cd plugins/versionpress && composer install )
+
+before_script:
+  - sudo /etc/init.d/mysql stop # Travis runs mysql by default and our docker tests want the mysql port (3306), so stop mysql
 
 script:
-    cd plugins/versionpress && composer install && ./vendor/bin/phpcs --standard=ruleset.xml
+  - set -e
+  - npm run lint:markdown
+  - ( cd plugins/versionpress && ./vendor/bin/phpcs --standard=ruleset.xml )
```

Comment:
This could be a `package.json` script as well, to be consistent with the rest. I'd call it `phpcs` or `lint:php`.

Reply:
done!

Reply:
👍